src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -476,6 +478,15 @@ private void openapi(RoutingContext routingContext) {
         });
     }
 
+    private void version(RoutingContext routingContext) {
```

Comment:
it should be `info` not only `version` anymore.

Reply:
Oh... I messed up a rebase somehow :(